File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.java
import java.util.Scanner;

public class Main
{
    public static void main(String[] args)
    {
        Scanner scanner = new Scanner(System.in);
        String[] tmp = scanner.nextLine().split("\\s");

        int[] bag = new int[Integer.parseInt(tmp[0])];
        for (int idx = 0; idx < Integer.parseInt(tmp[0]); idx++)
        {
            bag[idx] = idx + 1;
        }

        for (int idx = 0; idx < Integer.parseInt(tmp[1]); idx++)
        {
            String[] tmp_ = scanner.nextLine().split("\\s");
            int idx_1 = Integer.parseInt(tmp_[0]) - 1;
            int idx_2 = Integer.parseInt(tmp_[1]) - 1;

            int val = bag[idx_1];
            bag[idx_1] = bag[idx_2];
            bag[idx_2] = val;
        }

        for (int idx = 0; idx < bag.length; idx++)
        {
            System.out.print(bag[idx] + " ");
        }
    }
}

File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.py
bag_num, cnt = map(int, input().split())
bag = [(i+1) for i in range(bag_num)]

for idx in range(cnt):
    tmp_1, tmp_2 = map(int, input().split())
    bag[tmp_1-1], bag[tmp_2-1] = bag[tmp_2-1], bag[tmp_1-1]

for val in bag:
    print(val, end=' ')